MaterialLib/TwoPhaseModels/TwoPhaseFlowWithPPMaterialProperties.cpp

```diff
@@ -39,31 +40,37 @@ TwoPhaseFlowWithPPMaterialProperties::TwoPhaseFlowWithPPMaterialProperties(
     std::vector<std::unique_ptr<MaterialLib::PorousMedium::Porosity>>&&
         porosity_models,
     std::vector<std::unique_ptr<MaterialLib::PorousMedium::Storage>>&&
-        storage_models)
-    : _has_material_ids(has_material_ids),
+        storage_models,
+    std::vector<std::unique_ptr<
+        MaterialLib::PorousMedium::CapillaryPressureSaturation>>&&
```

Comment:
I think r-value variables includes the "MaterialID" properties, therefore they holds as vector